app/navigation/Navigator.scala

```diff
@@ -92,4 +95,11 @@ class Navigator @Inject()() {
     }
   }
 
+  private def produceApiHowToCreateNextPage(mode: Mode)(userAnswers: UserAnswers): Call = {
+    (mode, userAnswers.get(ProduceApiHowToCreatePage)) match {
+      case (_, Some(Editor)) => controllers.myapis.produce.routes.ProduceApiEnterOasController.onPageLoad(mode)
+      case (_, Some(Upload)) => routes.JourneyRecoveryController.onPageLoad()
+      case _ => routes.JourneyRecoveryController.onPageLoad()
```

Comment:
Both lines seem to lead to `routes.JourneyRecoveryController.onPageLoad()`
```suggestion
      case _ => routes.JourneyRecoveryController.onPageLoad()
```

Reply:
`case (_, Some(Upload))` will get a different implementation when we add the upload page. This is fine for a temporary implementation.